notebooks/official/explainable_ai/sdk_custom_image_classification_online_explain.ipynb

```diff
@@ -75,26 +77,26 @@
       "source": [
         "### Objective\n",
         "\n",
-        "In this tutorial, you learn how to use `Vertex AI Training and Explainable AI` to create a custom image classification model with explanations, and then you learn to use `Vertex AI Prediction` to make an online prediction request with explanations. You can alternatively create custom models using `gcloud` command-line tool or online using Cloud Console.\n",
+        "In this tutorial, you learn how to use Vertex AI training and Explainable AI to create a custom image classification model with explanations. Then, you learn how to use Vertex AI online prediction service to make online prediction requests with explanations. You can alternatively create custom models using `gcloud` command-line tool or online using Cloud Console.\n",
```

Comment:
Suggested edit:

- "Then, you learn how to use the Vertex AI online prediction service to make online prediction requests with explanations. Alternatively, you can create custom models..."